requirements.txt

```diff
@@ -1,6 +1,5 @@
-cherrypy==18.6.1
+elasticsearch[async] ~= 7.8.0
```

Comment:
I forgot to mention in the commit message, but the bump from 7.7 to 7.8 is significant (that's when the async API was added)